config/docdb/config.go

```diff
@@ -20,6 +27,23 @@ func Configure(p *config.Provider) {
 			}
 			return conn, nil
 		}
+		desc, _ := comments.New("If true, the password will be auto-generated and"+
+			" stored in the Secret referenced by the passwordSecretRef field.",
```

Comment:
nit: how about the description be as follows?

```suggestion
		desc, _ := comments.New("If true, the password will be auto-generated and"+
			" stored in the Secret referenced by the masterPasswordSecretRef field.",
```

Reply:
@turkenf updated, thanks for the catch!